README.md

```diff
@@ -19,6 +19,12 @@ You can see the extension settings popup by clicking on its icon.
 
 ![](img/settings.png)
 
+Example settings:
+```
+https://leanprover.zulipchat.com
+https://github.com/leanprover-community/mathlib
+```
+
```

Comment:
Oh, because of the bug you experienced, the extension didn't load up the default settings.

Having it fixed now, this is no longer needed 👍🏼